### PR TITLE
翻譯 scheduling.md

### PR DESCRIPTION
**Pull request by absszero**

### CODE REVIEW

File: scheduling.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+    - [避免任務重複](#preventing-task-overlaps)
+- [任務輸出](#task-output)
+- [任務掛勾](#task-hooks)
 
 <a name="introduction"></a>
-## Introduction
+## 簡介
 
-In the past, developers have generated a Cron entry for each task they need to schedule. However, this is a headache. Your task schedule is no longer in source control, and you must SSH into your server to add the Cron entries. The Laravel command scheduler allows you to fluently and expressively define your command schedule within Laravel itself, and only a single Cron entry is needed on your server.
+在過去，開發者必須為每個需要排程的任務產生 Cron 項目。然而令人頭疼的是任務排程不受版本控制，並且你需要 SSH 到你的伺服器增加 Cron 項目。Laravel 指令調度器（command scheduler）允許你清楚流暢的在 Laravel 當中定義指令排程，並且僅需要在你的伺服器上增加一條 Cron 項目即可。
```
Comment (jigsawye) [inline scheduling.md]: `scheduler` 翻譯成**排程器**會不會比較好？

**Comment by absszero** [comment]
調度器改為排程器

**Comment by jigsawye** [comment]
@absszero please resolve the conflicts, thanks!

**Comment by chentsulin** [comment]
superseded by #258

**Pull request closed without merging**